src/repositories/ColorRepository.ts

```diff
@@ -90,7 +89,7 @@ export function mapFrontifyColorCreateToColorApiCreate(color: FrontifyColorCreat
     };
 }
 
-export function mapFrontifyColorPatchToColorApiPatch(color: FrontifyColorPatch): ColorApiPatch {
+export function mapFrontifyColorPatchToColorApiPatch(color: ColorPatch): ColorApiPatch {
```

Comment:
Should update method name as well:
```suggestion
export function mapColorPatchToColorApiPatch(color: ColorPatch): ColorApiPatch {
```